index.js

```diff
@@ -482,10 +483,12 @@ program
               let nonIgnoredFilePaths = [];
               let files = filePaths.map((name, i) => {
                 let nameWithoutExt = name.slice(0, -path.extname(name).length);
+                // Formats rootDir/appsscript.json to appsscript.json. Preserves subdirectory names in rootDir
+                let formattedName = rootDir ? formattedName.slice(rootDir.length + 1, formattedName.length) : nameWithoutExt;
```

Comment:
Please wrap this long line.

Reply:
How do you declare `formattedName` and use it in the same statement? Do you mean `nameWithoutExt`? Maybe I'm missing something.